application/settings/llm_config_manager.py

```
@@ -216,11 +227,17 @@ def _apply_to_env(self, profile: LLMConfigProfile) -> None:
         env["WRITING_MODEL"] = writing
         env["SYSTEM_MODEL"] = system
 
+    def _invalidate_caches(self) -> None:
         try:
             from interfaces.api.dependencies import get_background_task_service
             get_background_task_service.cache_clear()
         except Exception:
             pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)